kuma/wiki/templates/wiki/create.html

```diff
@@ -53,6 +53,7 @@
           {% endif %}
         </ul>
         <input type="hidden" name="parent_topic" value="{{ parent_id | safe }}" />
+        {# FIXME check what's parentSlugPrefix doing here #}
```

Comment:
``git log -SparentSlugPrefix`` finds: 80302fec85a9ae60963d50a246733f2d3869e66d (fix bug 753584).  My guess is some client-slide form validation was not implemented.